renovate.json

```diff
@@ -39,5 +39,8 @@
   },
   "npm":{
     "addLabels": ["lang: nodejs"]
+  },
+  "kubernetes": {
+    "fileMatch": ["\\.yaml$"]
```

Comment:
**Suggestion:**
We should ignore `release/` and `kustomize/base/` since these folders are only modified when we create new released.

Reply:
We can use [`ignorePath`](https://docs.renovatebot.com/configuration-options/#ignorepaths).

Reply:
Good point, just updated it accordingly.